app/services/etherdelta_observer.py

```diff
@@ -41,14 +43,36 @@
 async def on_disconnect(io_client, event):
     logger.info("ED API client disconnected from %s", io_client.ws_url)
 
+
+
 async def process_orders(orders):
     not_deleted_filter = lambda order: "deleted" not in order or not order["deleted"]
     logger.info("Processing %i orders", len(orders))
     orders = list(filter(not_deleted_filter, orders))
     logger.debug("Filtered orders: %i", len(orders))
-
+    v = OrderMessageValidatorEtherdelta()
+    warnings.filterwarnings("ignore", category=DeprecationWarning)     
```

Comment:
Does this actually help?

Reply:
Oh rats... Forgot to delete that line. It does not help, so I deleted the "import warnings", but forgot this line.

Reply:
Ok, this definitely breaks things now.